stage1/usr_from_kvm/qemu.mk

```diff
@@ -0,0 +1,75 @@
+$(call setup-stamp-file,QEMU_STAMP)
+QEMU_TMPDIR := $(UFK_TMPDIR)/qemu
+QEMU_SRCDIR := $(QEMU_TMPDIR)/src
+QEMU_BINARY := $(QEMU_SRCDIR)/x86_64-softmmu/qemu-system-x86_64
+QEMU_BIOS_BINARIES := bios-256k.bin \
+    kvmvapic.bin \
+    linuxboot.bin \
+    vgabios-stdvga.bin \
+    efi-virtio.rom
+
+QEMU_CONFIGURATION_OPTS := --disable-bsd-user --disable-docs --disable-guest-agent --disable-guest-agent-msi \
+    --disable-sdl --disable-gtk --disable-vte --disable-curses --disable-cocoa --disable-brlapi --disable-vnc \
+    --disable-seccomp --disable-curl --disable-bluez --disable-cap-ng --disable-rbd --disable-libiscsi \
+    --disable-libnfs --disable-smartcard --disable-libusb --disable-glusterfs --disable-archipelago --disable-vhdx \
+    --disable-tcmalloc --disable-jemalloc --disable-debug-info --enable-virtfs --static --target-list=x86_64-softmmu
+QEMU_ACI_BINARY := $(S1_RF_ACIROOTFSDIR)/qemu
+
+# Using 2.6.0 stable release from official repository
+QEMU_GIT := git://git.qemu-project.org/qemu.git
+QEMU_GIT_COMMIT := v2.6.0-rc5
+
+$(call setup-stamp-file,QEMU_BUILD_STAMP,/build)
+$(call setup-stamp-file,QEMU_BIOS_BUILD_STAMP,/bios_build)
+$(call setup-stamp-file,QEMU_CONF_STAMP,/conf)
+$(call setup-stamp-file,QEMU_CLONE_STAMP,/clone)
+$(call setup-stamp-file,QEMU_DIR_CLEAN_STAMP,/dir-clean)
+$(call setup-filelist-file,QEMU_DIR_FILELIST,/dir)
+$(call setup-clean-file,QEMU_CLEANMK,/src)
+
+S1_RF_SECONDARY_STAMPS += $(QEMU_STAMP)
+S1_RF_INSTALL_FILES += $(QEMU_BINARY):$(QEMU_ACI_BINARY):-
+INSTALL_DIRS += \
+    $(QEMU_SRCDIR) :- \
+    $(QEMU_TMPDIR) :-
+
+# Bios files needs to be removed (source will be removed by QEMU_DIR_CLEAN_STAMP)
+$(foreach bios, $(QEMU_BIOS_BINARIES), $(eval CLEAN_FILES += $(S1_RF_ACIROOTFSDIR)/${bios}))
+
+$(call generate-stamp-rule,$(QEMU_STAMP),$(QEMU_CLONE_STAMP) $(QEMU_CONF_STAMP) $(QEMU_BUILD_STAMP) $(QEMU_ACI_BINARY) $(QEMU_BIOS_BUILD_STAMP) $(QEMU_DIR_CLEAN_STAMP),,)
+
+$(QEMU_BINARY): $(QEMU_BUILD_STAMP)
+
+# Copy bios files. Has to be done after making directories (UFK_CBU_STAMP)
```

Comment:
Also, question is why it needs to happen after UFK_CBU_STAMP. If it is only about the S1_RF_ACIROOTFSDIR, then S1_RF_INSTALL_FILES will make sure it is created before copying happens.